terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,11 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+max_task_count = 8
+# service configs
+use_set_environment_files = true
```

Comment:
This line can be removed by changing the default bool var

Reply:
thanks @Ngriffiths1 I have removed it.